app/src/main/java/com/duckduckgo/app/email/ui/EmailProtectionNotSupportedFragment.kt

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) 2021 DuckDuckGo
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.duckduckgo.app.email.ui
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import com.duckduckgo.app.browser.databinding.FragmentEmailProtectionNotSupportedBinding
+
+class EmailProtectionNotSupportedFragment : EmailProtectionFragment() {
+
+    private var _binding: FragmentEmailProtectionNotSupportedBinding? = null
+    private val binding get() = _binding!!
+
+    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View {
+        _binding = FragmentEmailProtectionNotSupportedBinding.inflate(inflater, container, false)
+        return binding.root
+    }
+
+    override fun onDestroyView() {
+        super.onDestroyView()
+        _binding = null
+    }
```

Comment:
This can be removed by using the `FragmentViewBindingDelegate`

Reply:
Will do! Thanks!